src/base/bittorrent/trackerentry.cpp

```diff
@@ -28,6 +28,33 @@
 
 #include "trackerentry.h"
 
+#include <QList>
+#include <QVector>
+
+QVector<BitTorrent::TrackerEntry> BitTorrent::parseTrackerEntries(const QStringView view)
+{
+    const QList<QStringView> trackers = view.split(u'\n');  // keep the empty parts to track tracker tier
+
+    QVector<BitTorrent::TrackerEntry> entries;
+    entries.reserve(trackers.size());
+
+    int trackerTier = 0;
```

Comment:
Won't users expect the new tracker tiers to start after the currently highest tier?
Just an observation, i have absolutely no idea what users want and i shouldn't be considered a sample on this.

Reply:
I'm not sure... any way this PR doesn't modify the behavior and we can keep an eye on users feedback.